pilot/pkg/model/validation_test.go

```diff
@@ -2606,6 +2606,38 @@ func TestValidateVirtualService(t *testing.T) {
 				}},
 			}},
 		}, valid: false},
+		{name: "multi short names", in: &networking.VirtualService{
+			Hosts: []string{"svc1", "svc2"},
```

Comment:
we dont allow shortnames at all do we? @frankbu ?

Reply:
For k8s service, i think we should only allow one service name specified.  For mesh external services, this fields must be DNS domains?